torch_geometric/distributed/local_graph_store.py

```diff
@@ -51,6 +58,48 @@ def _remove_edge_index(self, edge_attr: EdgeAttr) -> bool:
     def get_all_edge_attrs(self) -> List[EdgeAttr]:
         return [self._edge_attr[key] for key in self._edge_index.keys()]
 
+    # starting the partition info related to graph
+
+    def set_num_partitions(self, num_partitions: int) -> bool:
+        self.num_partitions = num_partitions
+        return True
+
+    def set_partition_idx(self, partition_idx: int) -> bool:
+        self.partition_idx = partition_idx
+        return True
+
+    def set_node_pb(
+            self, node_pb: Union[torch.Tensor, Dict[NodeType,
+                                                    torch.Tensor]]) -> bool:
+        self.node_pb = node_pb
+        return True
+
+    def set_edge_pb(
+            self, edge_pb: Union[torch.Tensor, Dict[NodeType,
+                                                    torch.Tensor]]) -> bool:
+        self.edge_pb = edge_pb
+        return True
+
+    def set_partition_meta(self, partition_meta: Dict) -> bool:
+        self.meta = partition_meta
+        return True
```

Comment:
* Why do these functions return `bool`?
* We should either define setters and have private instance variables or just use the public instance variables. No need to have setters on public instance variables.

Reply:
the current style is aligned with _edge_attr and _edge_id and all to be set later by set_xxx(). The bool return is also the same as from the original set_edge_index().

Reply:
I don't think we need that here. These are just instance variables, and don't need setters and getters as they are not private and these setters/getters do not perform any additional mutation. Would be good to clean that up.